frontend/privacypolicy.html

```diff
@@ -0,0 +1,92 @@
+<!-- <!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Privacy Policy</title>
+
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Kode+Mono:wght@400..700&display=swap" rel="stylesheet">
+
+	<script type="module" src="./static/js/index.js" defer></script>
+	<link rel="stylesheet" href="./static/css/styles.css">
+</head> -->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Privacy Policy</title>
+    <style>
+        body {
+            font-family: Arial, sans-serif;
+            line-height: 1.6;
+            margin: 20px;
+        }
+        h1, h2, h3 {
+            color: #333;
+        }
+        p {
+            margin-bottom: 20px;
+        }
+    </style>
+</head>
+<body>
+
+    <h1>Privacy Policy</h1>
+    <p>Last updated: 15.06.2024</p>
+
+    <p>Welcome to The Pong Game! This Privacy Policy explains how this project collects, uses, discloses, and protects your information when you visit and use our game page (the "Service").</p>
+
+    <p>By using the Service, you agree to the collection and use of information in accordance with this policy.</p>
+
+    <h2>1. Information We Collect</h2>
+
+    <p>We may collect and process the following types of information:</p>
+
+    <h3>Login Information</h3>
+    <p>Information necessary to access the game, such as:</p>
+    <ul>
+        <li>Username</li>
+        <li>Password</li>
+    </ul>
+
+    <h3>Friends Information</h3>
+    <p>Details about friends within the game, including:</p>
+    <ul>
+        <li>Friend lists</li>
+        <li>Friend interactions, ex. pending requests</li>
+    </ul>
+
+    <h3>Game Stats</h3>
+    <p>Information related to your gameplay, including:</p>
+    <ul>
+        <li>Scores</li>
+        <li>Ranking</li>
+        <li>Tournament information</li>
+    </ul>
+
+    <h2>2. How We Use Your Information</h2>
+
+    <p>We use the collected data for the following purposes:</p>
+    <ul>
+        <li>To provide and maintain the Service</li>
+        <li>To allow you to participate in interactive features of our Service</li>
+        <li>To track your game progress and achievements</li>
+        <li>To manage friendships within the game</li>
+    </ul>
+
+    <h2>3. Your Data Protection Rights</h2>
+
+    <p>Under the General Data Protection Regulation (GDPR), you have the following rights regarding your data:</p>
+    <ul>
+        <li><strong>The right to access</strong> – You have the right to request copies of your personal data.</li>
```

Comment:
Does this mean that we need them to be able to get a table/ file of all their data? Similar as like instagram/snapchat etc?

Reply:
Yep in theory. But in reality I think it's enough that we have thought about this, and have a way for the user to contact. So that we have taken in to account that GDPR is valid, but because we don't really save any data longer than our containers are up...